code/modules/mob/living/simple_animal/aquatic/aquatic_fish_lantern.dm

```diff
@@ -0,0 +1,26 @@
+/mob/living/simple_animal/aquatic/fish/large/lantern
+	name = "lantern fish"
+	desc = "An oily, glowing fish with a habit of nipping people. They are sometimes caught in cave rivers, and are rumoured to have cousins in the deep ocean."
```

Comment:
technically I think the original joke with the lanternfish was that it was the supposed explanation for someone being inexplicably covered in oil (because they'd actually fallen into the slaughterhouse water pit, which mixed their lantern's oil with the water. this then set them on fire

basically: it would be really funny if they got oil on you when they attack you